Sources/NIO/SocketAddresses.swift

```diff
@@ -264,11 +264,7 @@ public enum SocketAddress: CustomStringConvertible {
             throw SocketAddressError.unixDomainSocketPathTooLong
         }
 
-#if os(Android) // in Android first byte must be zero to use abstract namespace
-        let pathBytes = [0] + Array(unixDomainSocketPath.utf8) + [0]
```

Comment:
I don't know what this was ever about: a couple tests fail with an empty UDS path with this used, but pass after this pull.